### PR TITLE
Fixed conventions

### PR DESCRIPTION
**Pull request by dgarcia360**
Replaced some conventions:

*  API.
* Peer.
* Base32.
* Periods.
* MongoDB.

### CODE REVIEW

File: whitepaper/chapters/accounts.tex [hunk 1]
```diff
@@ -13,9 +13,9 @@ \subsection{Addresses}
 
 An \nind{address} is a base32\footnote{ \url{http://en.wikipedia.org/wiki/Base32} } encoded triplet consisting of:
 \begin{itemize}
-	\item{network byte}
-	\item{160-bit hash of the account's public key}
-	\item{4 byte checksum}
+	\item{network byte.}
+	\item{160-bit hash of the account's public key.}
+	\item{4 byte checksum.}
```
Comment (Jaguar0625) [inline whitepaper/chapters/accounts.tex]: if you're adding periods, should you capitalize first word?

File: whitepaper/chapters/accounts.tex [hunk 2]
```diff
@@ -13,9 +13,9 @@ \subsection{Addresses}
 
 An \nind{address} is a base32\footnote{ \url{http://en.wikipedia.org/wiki/Base32} } encoded triplet consisting of:
```
Comment (Jaguar0625) [inline whitepaper/chapters/accounts.tex]: Base32?

File: whitepaper/chapters/unconfirmed.tex [hunk 3]
```diff
@@ -15,8 +15,8 @@ \section{Unconfirmed Transactions}
 \begin{enumerate}
 	\item{A client sends a new transaction directly to the node.}
```
Comment (Jaguar0625) [inline whitepaper/chapters/unconfirmed.tex:16]: i am not sure if these should lead with articles "A"

File: whitepaper/chapters/consensus.tex [hunk 4]
```diff
@@ -200,8 +200,8 @@ \subsection{Sybil Attack}
 Consider two extremes:
 
 \begin{enumerate}
-	\item{The attacker has a single account with $N \cdot \mu$ currency}
-	\item{The attacker has $N$ accounts with $\mu$ currency}
+	\item{The attacker has a single account with $N \cdot \mu$ currency.}
+	\item{The attacker has $N$ accounts with $\mu$ currency.}
```
Comment (Jaguar0625) [inline whitepaper/chapters/consensus.tex:204]: I am unsure about leading articles "The"

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]

File: whitepaper/chapters/accounts.tex (hunk 1 above)
Comment (dgarcia360) [reply whitepaper/chapters/accounts.tex]: In the latest commit, bullet lists with uncomplete sentences are written without the period.

File: whitepaper/chapters/unconfirmed.tex (hunk 3 above)
Comment (dgarcia360) [reply whitepaper/chapters/unconfirmed.tex:16]: I would keep them, makes the sentence complete and easier to read.

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**